build/__dotNetCoreSdkVersions.sh

```diff
@@ -10,3 +10,4 @@ DOT_NET_50_SDK_VERSION='5.0.408'
 DOT_NET_60_SDK_VERSION='6.0.401'
 DOT_NET_60_RTM_SDK_VERSION='6-0-100-rtm-21527-11'
 DOT_NET_70_SDK_VERSION='7.0.100-rc.2.22477.23'
+RUNTIME_VERSIONS=("1.0" "1.1" "2.0" "2.1" "2.2" "3.0" "3.1" "5.0" "6.0" "7.0")
```

Comment:
Could you please clarify where this (and the similar one for other languages) is used?

Reply:
Adding support for lists in constants.yaml to be outputted to shell scripts added this: see https://github.com/microsoft/Oryx/pull/1643#discussion_r1009494706

Basically, I needed to be able to create a list of dotnet versions that we want to upload to a staging repo, and unfortunately adding that functionality meant that these lists from the constants.yaml are also created. But per that comment, i don't see them as being much of an issue since they are never used

Reply:
Ended up excluding these!